src/lib.rs

```diff
@@ -187,16 +202,17 @@ extern crate serde_derive;
 pub use join::{Join, JoinIter, JoinParIter, ParJoin};
 pub use shred::{Dispatcher, DispatcherBuilder, Fetch, FetchId, FetchIdMut,
                 FetchMut, RunNow, RunningTime, System, SystemData};
-
-#[cfg(not(target_os = "emscripten"))]
-pub use shred::{AsyncDispatcher};
-
 pub use storage::{BTreeStorage, CheckStorage, DenseVecStorage, DistinctStorage, FlaggedStorage,
                   HashMapStorage, InsertResult, NullStorage, ReadStorage, Storage,
                   UnprotectedStorage, VecStorage, WriteStorage};
 pub use world::{Component, CreateIter, CreateIterAtomic, EntitiesRes, Entity, EntityBuilder,
                 Generation, LazyUpdate, World};
 
+#[cfg(not(target_os = "emscripten"))]
+pub use shred::{AsyncDispatcher};
```

Comment:
Sorry, I must have made a mistake while copy-pasting sections around during the rebase.